src/Altinn.AccessManagement.Core/Models/RightSource.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Text.Json.Serialization;
+using Altinn.AccessManagement.Core.Enums;
+
+namespace Altinn.AccessManagement.Core.Models
+{
+    /// <summary>
+    /// This model describes a source from where a right exist for a user or party
+    /// </summary>
+    public class RightSource
+    {
+        /// <summary>
+        /// Gets or sets the set of type of source this right originated from (Role, AccessGroup, AppDelegation, ResourceRegistryDelegation etc.)
+        /// </summary>
+        [JsonConverter(typeof(JsonStringEnumConverter))]
+        public RightSourceType RightSourceType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the unique identifier for the specific policy the right originates (Output only).
+        /// </summary>
+        public string PolicyId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the version of the policy which the right originates (Output only).
+        /// </summary>
+        public string PolicyVersion { get; set; }
+
+        /// <summary>
+        /// Gets or sets the unique identifier for the specific rule within the policy the right originates (Output only).
+        /// </summary>
+        public string RuleId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the party offering the rights to the receiving (CoveredBy) entity.
+        /// </summary>
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        public int OfferedByPartyId { get; set; } // Todo: remove?
+
+        /// <summary>
+        /// Gets or sets the party receiving (covered by) the rights from the delegating (OfferedByPartyId) entity
+        /// </summary>
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        public List<AttributeMatch> CoveredBy { get; set; } // Todo: remove?
```

Comment:
Er det noe i output noe sted som indikerer hvem den faktiske brukeren/orgen som får den effektive rettigheten (den det er spurt om). Denne er vel bare hvem som rettigheten er gitt til

Reply:
I have removed the CoveredBy property for now.

For delegation policy right sources the CoveredBy is identical to the UserSubjects, and for App/Resource-Policy sources we don't get the CoveredBy dimension from the Role lookup in Altinn 2.0

But to answer your question: no, the model currently don't specify anything related to who the "To" user is.
I assumed since the "To" user is required input it's implicitly known.
(also support for lookup of org/party rights is not implemented currently, but should be an easy extension later by skipping the role and keyroleunit lookups)

Reply:
The rights query operation now only returns a list of Right models. Is your suggestion that we instead have a RightsQueryResponse model which (in addition to the rights list) contains more of the query metadata about the From/To parties such as: SSN/OrgNo, Name, KeyRoleUnits, MainUnits etc?

Reply:
That would make it possible to build a complete picture based on the response alone I guess.
E.g. When a RightSource.UserSubjects points to a PartyId you can find this in partyId in the To partys KeyRoleUnits, to see who actually was the recipient.

Reply:
I'll revisit this as I move forward with making the query for delegable rights